src/main/resources/assets/mo/lang/lol_US.lang

```diff
@@ -0,0 +1,62 @@
+##########################################################################################################
+# Items
+
+item.matter_scanner.name=MATTR SCANNR
+item.emergency_ration.name=EMERGENCY RASHUN
+item.phaser.name=FASR
+item.battery.name=BATTERY
+item.creative_battery.name=CREATIV BATTERY
+item.me_conversion_matrix.name=MATTR-ENERGY CONVERSHUN MATRIX
+item.isolinear_circuit.mk1.name=ISOLINEAR CIRCUIT MK1
+item.isolinear_circuit.mk2.name=ISOLINEAR CIRCUIT MK2
+item.isolinear_circuit.mk3.name=ISOLINEAR CIRCUIT MK
+item.isolinear_circuit.mk4.name=ISOLINEAR CIRCUIT MK4
+item.upgrade.base.name=UPGRADE SHELL
+item.h_compensator.name=HEISENBERG COMPENSATOR
+item.integration_matrix.name=INTEGRASHUN MATRIX
+item.s_magnet.name=Superconductor MAGNET
+item.matter_dust_refined.name=REFIND MATTR DUST
+item.matter_dust.name=MATTR DUST
+item.matter_dust.details=PUT IN RECYCLR 2 REFINE
+item.machine_casing.name=MACHINE CASIN
+item.dilithium_crystal.name=DILITHIUM CRYSTAL
```

Comment:
No new name for this either?

Reply:
They are spelled slightly differently, and in all caps.

Reply:
Not this one: item.dilithium_crystal.name=DILITHIUM CRYSTAL